go-controller/pkg/config/utils.go

```diff
@@ -104,7 +116,15 @@ func ParseFlowCollectors(flowCollectors string) ([]HostPort, error) {
 		if err != nil {
 			return nil, fmt.Errorf("collector port %s is not a valid port: %v", port, err)
 		}
-		parsedFlowsCollectors = append(parsedFlowsCollectors, HostPort{Host: ipp, Port: int32(parsedPort)})
+		// checking if HostPort entry is duplicate
```

Comment:
On the other side, the key that is used is the `String()` output of the IP **after it has been parsed to binary** from the original text. In that case, I verified that IP addresses that were passed with a different notation resulted in the same key (because both keys are built from equal binary notations).

Reply:
Please refer to this test: https://github.com/ovn-org/ovn-kubernetes/pull/2745/files#diff-9b81845552f4b8f27fc0d60a4bcaaa165b3f11135a92739a764c2f1655a40d94R265